### PR TITLE
[Helm] Give all users projectCreator role at the cluster level

### PR DESCRIPTION
**Pull request by albscui**

### CODE REVIEW

File: etc/helm/pachyderm/values.yaml [hunk 1]
```diff
@@ -523,7 +523,9 @@ pachd:
   #   - repoReader
   #   robot:eve:
   #   - repoWriter
-  pachAuthClusterRoleBindings: {}
+  pachAuthClusterRoleBindings:
+    allClusterUsers:
+      - projectCreator # by default, everyone should be able to create projects
```
Comment (molinamelendezj) [inline etc/helm/pachyderm/values.yaml:528]: No need for projectOwner?

**Review by molinamelendezj — APPROVED** [review]

File: etc/helm/pachyderm/values.yaml [hunk 2]
```diff
@@ -523,7 +523,9 @@ pachd:
   #   - repoReader
   #   robot:eve:
   #   - repoWriter
-  pachAuthClusterRoleBindings: {}
+  pachAuthClusterRoleBindings:
```
Comment (acohen4) [inline etc/helm/pachyderm/values.yaml:526]: I forgot that we currently ship empty role bindings. I think this is ok - but if a user is currently activating auth via git ops, but is setting cluster role bindings via API, we will be overwriting those. I think we need to make sure this is what we want

File: etc/helm/pachyderm/values.yaml (hunk 1 above)
Comment (albscui) [reply etc/helm/pachyderm/values.yaml:528]: No need for projectOwner. We want users to create projects, and become owners, but don't want to automatically make everyone owner by default.

**Comment by albscui** [comment]
Closed in favor of: https://github.com/pachyderm/pachyderm/pull/8791

**Pull request closed without merging**